### PR TITLE
Added meaningfull random sentences.

### PR DESCRIPTION
**Pull request by Ayush4web**

### CODE REVIEW

File: assets/js/practice.mjs [hunk 1]
```diff
@@ -34,50 +36,31 @@ let startTime = Date.now();
...
-    for (let i = 0; i < quoteLength; i++) {
-        words += makeword(length, charactersLength, characters) + " ";
-    }
-    words_length = words.length;
-    return words;
+  
+
+    
+   
+    return makeSentence(selectedDifficultyLevel);
```
Comment (arpitghura) [inline assets/js/practice.mjs:63]: Not generating enough number of words.

File: assets/js/practice.mjs [hunk 2]
```diff
@@ -106,7 +89,8 @@ const startTimer = (time) => {
 
 // start the typing game
 startBtn.addEventListener("click", () => {
-    const quote = makequote();
+    const quote = makequote()
+
```
Comment (arpitghura) [inline assets/js/practice.mjs:93]: At line 141 change the last calculation `(extracted_words.length - 2)` to `(extracted_words.length - 1)`

**Review by arpitghura — CHANGES_REQUESTED** [review]

File: assets/js/practice.mjs (hunk 1 above)
Comment (Ayush4web) [reply assets/js/practice.mjs:63]: You can add as many verbs you want to that

**Comment by arpitghura** [comment]
@Ayush4web If you are working on this PR then we can `convert it to draft` and when your task completes I will merge it.

**Comment by Ayush4web** [comment]
all changes done

**Review by arpitghura — APPROVED** [review]
Good work

**Pull request merged**